### PR TITLE
Builtthisweek

### LINKED ISSUE #232: Osr Built this week drilldown
replacement for spud btw detail

### PR DESCRIPTION
**Pull request by andrewfraser73**
Simple built this week drill down from OSR using the report components

Resolves #232

### CODE REVIEW

File: src/Persistence.LinnApps/ServiceDbContext.cs [hunk 1]
```diff
@@ -142,6 +142,8 @@ public class ServiceDbContext : DbContext
 
         private DbQuery<PtlMaster> PtlMasterSet { get; set; }
 
+        public DbQuery<BuiltThisWeekStatistic> BuiltThisWeekStatistics { get; set; }
```
Comment (richardiphillips) [inline src/Persistence.LinnApps/ServiceDbContext.cs]: public should be before the privates. Can we get rid of those private dbquery entries now we've got the single record repo?

File: src/Resources/BtwRequestResource.cs [hunk 2]
```diff
@@ -0,0 +1,6 @@
+﻿namespace Linn.Production.Resources
+{
+    public class BtwRequestResource {
+        public string CitCode { get; set; }
```
Comment (richardiphillips) [inline src/Resources/BtwRequestResource.cs]: could just be cit code request resource for reuseability?

File: tests/Integration/Service.Tests/BuiltThisWeekModuleSpecs/ContextBase.cs [hunk 3]
```diff
@@ -0,0 +1,59 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
```
Comment (richardiphillips) [inline tests/Integration/Service.Tests/BuiltThisWeekModuleSpecs/ContextBase.cs]: weird usings

File: src/Domain.LinnApps/BuiltThisWeekStatistic.cs [hunk 4]
```diff
@@ -0,0 +1,19 @@
+﻿namespace Linn.Production.Domain.LinnApps
```
Comment (richardiphillips) [inline src/Domain.LinnApps/BuiltThisWeekStatistic.cs]: there's a folder for view models in the domain. Perhaps this should go in it

File: src/Persistence.LinnApps/ServiceDbContext.cs (hunk 1 above)
Comment (andrewfraser73) [reply src/Persistence.LinnApps/ServiceDbContext.cs]: yes we should probably rewrite these - will raise an issue.

**Pull request merged**